src/test/java/org/zalando/riptide/AsyncTest.java

```diff
@@ -75,6 +75,20 @@ public void shouldCall() {
     }
 
     @Test
+    public void shouldCallWithoutParameters() {
```

Comment:
> Now we have the same test twice.

Once with a synchronous Rest, once with AsyncRest, yes.
I guess I can remove it from here if you don't deem it necessary to test that this works asynchronously, too.